<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cDistributorEditDialog</class>
 <widget class="QDialog" name="cDistributorEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distributor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpAddress"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Postal:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpPostalCode"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>City:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpCity"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Country:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpCountry"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Phone:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpPhone"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>EMail:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpEMail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Web:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_lpLink"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="m_lpDescription"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_lpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>cDistributorEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>cDistributorEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
